test/fixtures/flow/type-annotations/136/actual.js

```diff
@@ -0,0 +1,4 @@
+type A = {
+	p: {},
+	...{},
+};
```

Comment:
We should see if we can setup greenkeeper or a bot to update the package + yarn @fkling